src/app-logic/url-handling.js

```diff
@@ -381,38 +428,41 @@ export function stateFromLocation(
     profileSpecific: {
       implementation,
       lastSelectedCallTreeSummaryStrategy: toValidCallTreeSummaryStrategy(
-        query.ctSummary
+        query.ctSummary || undefined
       ),
-      invertCallstack: query.invertCallstack !== undefined,
-      showUserTimings: query.showUserTimings !== undefined,
-      showJsTracerSummary: query.summary !== undefined,
+      invertCallstack: query.invertCallstack === undefined ? false : true,
```

Comment:
Those changes look like the same, but flow complains on the previous versions. So this works 🤷‍♂

Reply:
🤷‍♂️